layout/page.ftd

```diff
@@ -37,6 +37,7 @@ ftd.length.px spacing: 0
 page-widths widths: $widths
 ;; todo: is the follow reference to `.gap` correct?
 ds.gap margin: $ds.spaces.vertical-gap.large
+optional ftd.background-image bg-image:
```

Comment:
Add background instead of `bg-image` so all kinds of backgrounds can be added (solid, gradient, image etc).